### ISSUE TITLE
Scans in json format are not reproducible

### ISSUE THREAD

**Issue by antnks** [MobSF/mobsfscan#30]
When scanning the same source tree multiple times and using json output format, the results each time are coming in different order which makes it impossible to diff.

Step to repro:
1. git clone https://github.com/google/ground-android
2. cd ground-android
3. for i in {1..20}; do mobsfscan --json . > ../$i.mobsf; done
4. cd ..
5. md5sum *.mobsf
6. some scan results will be different

```
$ md5sum *.mobsf
b75e43b0773432f489d8c55cf4842509  10.mobsf
32c969b69ebdd28c95bc3452a52eba25  11.mobsf
8e7aadfbea3c6ff6efb04b88e03976d3  12.mobsf
423a89fe1a9d490a654d87fca754125d  13.mobsf
a3eb93de089f21074d2c7f1722fd49d2  14.mobsf
29a22271ebef7ca6cfcd8d4d08cdc38c  15.mobsf
e5264346c343756960b584b77a2c218b  16.mobsf
bf0125fc3ff3dc34d4cc29f716942d9c  17.mobsf
587de9df5c6b3d0a5a64030e3276eb07  18.mobsf
b1ce4b7786838436968b030877b8b81a  19.mobsf
096b9e788ab5be9258a4fd4860d28638  1.mobsf
592f5cdef06cee9fc42b5a806885a134  20.mobsf
a835687e06dd18fd79c359a9a234b83d  2.mobsf
1156b46954cadc1cfb163ac6f6a761d8  3.mobsf
a7d33073936f5cf25a2f31b52bf0b55d  4.mobsf
587de9df5c6b3d0a5a64030e3276eb07  5.mobsf
5e888b6fd84bf1c156c04730a8806bee  6.mobsf
9b45bf1d81dc14cdfa520a68cd1f67a2  7.mobsf
47c5213631c96f20776888177d4040e8  8.mobsf
587de9df5c6b3d0a5a64030e3276eb07  9.mobsf
```

The issue is that findings are dumping inside `"files": [` array
The same set of issues found may come in different order causing the results to look different

Actual behavior: findings are numbered and may come in different order on each run

Expected behavior: reproducible scan, when multiple runs produce the same results in the same order

**Comment by ajinabraham** [comment]
I will see what can be done here.

**Comment by ajinabraham** [comment]
Should be fixed in libsast

**Comment by ajinabraham** [comment]
Fixed in libsast 1.5.2
